### ISSUE TITLE
Inference checkpoint not separating mixtures?

### ISSUE THREAD

**Issue by joeoct93** [lin9x/AV-Sepformer#4]
Hello, I wanted to test the AV-Sepformer's performance, but found that the checkpoint you provided doesn't really separate the audio mixtures. Is this a pretrained checkpoint that requires fine-tuning to get the results you showed in your demo? Or am I just implementing it wrong? I have gotten the visual embeddings from MuSE like instructed, and tested the model on the VoxCeleb2 test set. However, I found that while the conditions has very little effect. It does seem to change some values, but it's not an audible difference. If I didn't look at the actual values of the audio arrays, I would've thought that they were the same as the original mixture. I know that the checkpoint was loaded, because without the checkpoint, the AV-sepformer basically gives some static as pred_wav, and I know that the visual embeddings do give an impact, because I tried giving 0 as visual embedding, and found that they give tensors like following:
mixture: tensor([[ 0.0069, -0.0178,  0.0105,  ..., -0.0558, -0.0598, -0.0560],
                          [ 0.0069, -0.0178,  0.0105,  ..., -0.0558, -0.0598, -0.0560]])
pred_wav(with vis_emb): tensor([[ 0.0018, -0.0005,  0.0151,  ..., -0.0414, -0.0329, -0.0232],
                                                     [-0.0005, -0.0045,  0.0097,  ..., -0.0807, -0.0767, -0.0668]])
pred_wav(vis_emb=0): tensor([[-0.0038, -0.0043, -0.0028,  ..., -0.1554, -0.1534, -0.1383],
                                                  [-0.0038, -0.0043, -0.0028,  ..., -0.1554, -0.1534, -0.1383]])
As can be seen, the pred_wav(with vis_emb) is different from mixture and pred_wav(vis_emb=0) in that its tensors have different values in the first and second dimension. Even so, it still sounds pretty much the same as the mixture, except for the artifacts.
So, is this checkpoint one that requires further fine-tuningo on my part to get the results you showed in your demo? Or am I just implementing it wrong?

Nevermind, model.eval() works the way I described above, but it turns out having it in model.train() does the trick.
